src/app/ng-intercom/intercom/intercom.ts

```diff
@@ -43,17 +44,16 @@ export class Intercom {
    * If you'd like to control when Intercom is loaded, you can use the 'boot' method.
    * This is useful in situations like a one-page Javascript based application where the user may not be logged in
    * when the page loads. You call this method with the standard intercomSettings object.
-   * @param {object} [intercomData] Your intercom configuration
    */
-  boot(intercomData?: BootInput) {
+  public boot(intercomData?: BootInput): void {
     if (!isPlatformBrowser(this.platformId)) {
       return
     }
 
-    const data = {
+    const data = Object.assign({}, {
```

Comment:
This seems like a more verbose way to write what it already did. Is this Object.assign necessary?